site/src/main/scala/scalatex/site/Site.scala

```diff
@@ -97,10 +95,10 @@ trait Site{
   }
 
   def generateHtml(outputRoot: Path) = {
-    for((path, frag) <- content){
+    for((path, page) <- content){
       val txt = html(
-        head(headFrags),
```

Comment:
Shall we? I thought it could be used as kind of a default/base header to build upon. Maybe rename it?